serialization/src/main/scala/sbt/serialization/Using.scala

```diff
@@ -0,0 +1,19 @@
+package sbt.serialization
+
+import java.io.{ File, FileOutputStream, BufferedWriter, OutputStreamWriter }
+import java.nio.charset.Charset
+
+object Using {
```

Comment:
So we're mimicking Using from sbt.IO here?   I think you want this to be private for now.